# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1330135608 28800
#      Fri Feb 24 18:06:48 2012 -0800
# Node ID 7d5a653825b9450fb33ede6f8475a0a46141789a
# Parent  61cc4c715962a650740d24d0bc3d796280e7b028
profshow.m: Add more input validation tests

* profshow.m: Add more input validation tests

diff --git a/scripts/general/profshow.m b/scripts/general/profshow.m
--- a/scripts/general/profshow.m
+++ b/scripts/general/profshow.m
@@ -93,10 +93,12 @@ endfunction
 %!demo
 %! profile on;
 %! expm (rand (500) + eye (500));
 %! profile off;
 %! profshow (profile ("info"), 5);
 
 %!error profshow ()
 %!error profshow (1, 2, 3)
-%!error profshow (struct (), 1.2)
+%!error <N must be a positive integer> profshow (struct (), ones (2))
+%!error <N must be a positive integer> profshow (struct (), 1+i)
+%!error <N must be a positive integer> profshow (struct (), -1)
 
